client/components/mma/dataPrivacy/YourDataSection.tsx

```diff
@@ -13,6 +13,16 @@ interface YourDataSectionProps {
 	consents: ConsentOption[];
 	toggleConsent: ClickHandler;
 }
+
+const consentSubscribedValueInverter = (
```

Comment:
I would recommend trying to use the same inversion code from the [OptOutSection page](https://github.com/guardian/manage-frontend/blob/7bb57675d6c09ac4b7684bb1b7989abd31279b27/client/components/mma/identity/emailAndMarketing/OptOutSection.tsx#L16-L67) so that this very unobvious logic isn't duplicated in two places? Perhaps relevant inversion code could be pulled out as a util somehow?